src/components/Docs/components/ContractRegistryChains.tsx

```
@@ -0,0 +1,127 @@
+import { useEffect, useMemo, useState } from "react";
+
+import { LoadingTable, NetworkTypeTabs, networkTypeTabs } from "~/components/shared";
+import { NetworkType } from "~/lib/app.types";
+
+import { ethers } from "ethers";
+import CONTRACT_REGISTRY from "@zetachain/protocol-contracts/abi/Registry.sol/Registry.json";
+import { networkDetails } from "./NetworkDetails";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)